### ISSUE TITLE
Reducing complexity

### ISSUE THREAD

**Issue by pangoSE** [osmandapp/Osmand#9328]
Thanks for making a great piece of free software. I use it daily and love it. ❤

I have this idea of spreading OsmAnd to the general population here in Sweden.
I would really like to help people learn to navigate using it.
I do realize though that it is unfortunately quite complex.
I think this is important to work to bring down.

E.g. I never during 3 years used the markers menu. Sometimes markers appear and they are annoying and "hard" to get rid of because I don't know how they work. Could this function be deemed "advanced" and hidden for users who did not explicitly enable it (and got a proper introduction for how to use it)?

Could we create a separate introduction for each of the "advanced" features when enabled?

**Comment by pangoSE** [comment]
In case the markers menu is now not really needed for navigation route planning, would it make sense to remove it?

**Comment by westis** [comment]
I totally agree. The main reason that keeps me from using OsmAnd as my default navigation app is the complexity. Google Maps etc. is bare-stripped to only show the functionality that most users need. OsmAnd has an advantage in offering more advanced functionality, but much of it is just in the way for most users. It's better to only make such features available if the user enables them, while keeping the default UI as simple as possible.

**Comment by vshcherb** [comment]
That's a good idea to move markers into plugin. Though we will start with simplifying settings menu first.

**Comment by Aiurek** [comment]
@pangoSE check #5653 for additional info. Mby u got an idea to contribute

**Comment by pangoSE** [comment]
> @pangoSE check #5653 for additional info. Mby u got an idea to contribute

Sorry, I still don't understand how they work. Is there an explanation somewhere?

**Comment by vshcherb** [comment]
Markers is a simple point that you can mark on the map and it remains visible, it's much simpler than favorites cause you shouldn't give a name to it and it is considered to be a short-life point: easy to add, fast to remove.

**Comment by seniorm0ment** [comment]
@westis 
I agree here, OSMAnd is extremely powerful which I appreciate. But for certain circumstances it can be way too much. Google Maps feels very simple and straightforward, I'm not overwhelmed with map colors, or icons, the interface isn't obtrusive or make things feel clusterphobic.

In my opinion, this is important for driving because if you are spending more time trying to look at your phone understanding what the device is telling you, then it's less time your eyes are on the road.

There are many things OSMAnd does great. I want to be able to use it full time but I feel like every time I try to use it, it's just a bit of a hassle.

**Comment by sonora** [comment]
> **Reducing complexity (Markers)**
E.g. I never during 3 years used the markers menu.

Related: https://github.com/osmandapp/OsmAnd/issues/16513#issuecomment-1445051771
